lib/Common/Core/ConfigFlagsTable.cpp

```diff
@@ -405,9 +405,11 @@ namespace Js
     {
         const auto VerifyExecutionModeLimits = [this]()
         {
+CLANG_WNO_BEGIN("-Wunused-variable");
```

Comment:
Also thinking that these asserts and variables could be wrapped in #ifdef DEBUG instead. The warning could still be valid in debug build.